app/cni/pkg/install/utils.go

```diff
@@ -22,3 +28,17 @@ func parseBytesToHashMap(bytes []byte) (map[string]interface{}, error) {
 	}
 	return parsed, nil
 }
+
+func CreateNewLogger(name string, logLevel kuma_log.LogLevel) logr.Logger {
+	return core.NewLoggerTo(os.Stderr, logLevel).WithName(name)
+}
+
+func SetLogLevel(logger *logr.Logger, level string, name string) {
+	logLevel, err := kuma_log.ParseLogLevel(level)
+
+	if err != nil {
```

Comment:
why not return an error?

Reply:
If we want to quit if invalid level is set then yes we can return an error.

Reply:
If we want to quit the app if the wrong log level is set then we could do that.

Reply:
We do this in Kuma CP / other binaries and I think it makes sense. It protects the user from typos